src/test/java/org/apache/bcel/AbstractTestCase.java

```diff
@@ -63,9 +64,12 @@ public AnnotationEntryGen createFruitAnnotationEntry(final ConstantPoolGen cp,
 
     public SyntheticRepository createRepos(final String cpentry)
     {
-        final ClassPath cp = new ClassPath("target" + File.separator + "testdata"
-                + File.separator + cpentry + File.separator);
-        return SyntheticRepository.getInstance(cp);
+        try (ClassPath cp = new ClassPath("target" + File.separator + "testdata"
+                + File.separator + cpentry + File.separator)) {
+            return SyntheticRepository.getInstance(cp);
+        } catch (IOException e) {
+            throw new Error(e);
```

Comment:
This feels nasty IMO, why not use an `UncheckedIOException` here instead of an `Error`?

Reply:
Good idea.  I have done that.